core/src/main/resources/dokka/styles/style.css

```diff
@@ -427,6 +427,59 @@ tr.platform-tagged {
     flex: auto;
 }
 
+.platform-hinted {
+    display: block;
+}
+
+.platform-hinted > .platform-bookmarks-row {
+    display: flex;
+    flex-direction: row;
+    padding: 0px;
+}
+
+.platform-hinted > .platform-bookmarks-row > .platform-bookmark {
+    width: 64px;
+    height: 36px;
+
+    border: 2px solid white;
+
+    background: white;
+
+    outline: none;
+    flex: none;
+    order: 5;
+    align-self: flex-start;
+    margin: 0px 0px;
+}
+
+.platform-hinted > .platform-bookmarks-row > .platform-bookmark[index='js']:hover{
+    border-top: 2px solid orange;
+}
+
+.platform-hinted > .platform-bookmarks-row > .platform-bookmark[index='jvm']:hover{
+    border-top: 2px solid crimson;
+}
+
+.platform-hinted > .platform-bookmarks-row > .platform-bookmark[index='common']:hover{
+    border-top: 2px solid gray;
+}
+
+.platform-hinted > .platform-bookmarks-row > .platform-bookmark[index='native']:hover{
+    border-top: 2px solid blue;
+}
```

Comment:
Hardcoding colors for specific platforms in CSS is not a good idea IMHO.  We should have a class with a color for common elements and then some classes that we choose from depending on the order I think? Or even randomly